cmd/kubeadm/app/phases/certs/renewal/readwriter_test.go

```diff
@@ -239,7 +239,7 @@ func TestFileExists(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			if got := fileExists(tt.filename); got != tt.want {
+			if got, _ := fileExists(tt.filename); got != tt.want {
```

Comment:
after this change there is a missing unit test case for the `return false, err` return of `func fileExists(filename string) (bool, error) {` but i don't insist of adding it.

we could use os.Chmod() or a os.WriteFile with a restrictive FileMode, but then this test will not work on Windows, i think.

so fine to leave this test case out for now.